### ISSUE TITLE
[calendar] subscribe a CalDAV calendar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#641]
```
Token: 644951cc238bce6d2ef226ac1c945477 (Don't modify this token!)
Version: 17.0.12 (27-02-2014)
PHP: 5.3.10-1ubuntu3.9
RCMAIL: 0.9.5
Database: mysql
SERVER: Apache/2.2.22 (Ubuntu)
----
I.  Issue Description:
Error-Message: Please add or subscribe a CalDAV calendar in order to be able to 
modify calendar

II. Steps to reproduce the Issue:
1. Open Calendar -Tab
2. Push Button "Create Event"
```

Original issue reported on code.google.com by `CHericks...@gmail.com` on 4 Mar 2014 at 12:24

**Comment by GoogleCodeExporter** [comment]
```
Isn't this message clear?
```

Original comment by `roland.l...@gmail.com` on 4 Mar 2014 at 3:00
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Yes, but it's wrong.

Under Settings-Calendar the Default calendar label is "Standard", and under 
Settings-Calendar-Categories there is a entry called "Standard" and it's 
protected.

If I add a new entry under Settings-Calendar-Categories, then it works.
```

Original comment by `CHericks...@gmail.com` on 4 Mar 2014 at 3:47

**Comment by GoogleCodeExporter** [comment]
```
Is there a default caldav url in calendar settings?
```

Original comment by `roland.l...@gmail.com` on 4 Mar 2014 at 3:49

**Comment by GoogleCodeExporter** [comment]
```
No, I can not find this setting under Settings-Preferences-Calendar.
```

Original comment by `CHericks...@gmail.com` on 4 Mar 2014 at 5:11

Attachments:
- [Calendar.jpg](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-641/comment-4/Calendar.jpg)
- [categories.jpg](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-641/comment-4/categories.jpg)
- [Installation.jpg](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-641/comment-4/Installation.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Did you enable caldav_protect?
```

Original comment by `roland.l...@gmail.com` on 4 Mar 2014 at 5:51

**Comment by GoogleCodeExporter** [comment]
```
No, and "backend"="database"
```

Original comment by `CHericks...@gmail.com` on 5 Mar 2014 at 7:40

**Comment by GoogleCodeExporter** [comment]
```
I have the same issue after upgrading to 
calendar (17.0.12)
carddav (5.4.47)

```

Original comment by `gkapeta...@gmail.com` on 10 Mar 2014 at 10:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `roland.l...@gmail.com` on 10 Mar 2014 at 10:51
- Changed state: **Re-Opened**

**Comment by GoogleCodeExporter** [comment]
Original comment by `roland.l...@gmail.com` on 10 Mar 2014 at 2:02
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Confirm that it's fixed with calendar (17.0.14)

thanks
```

Original comment by `gkapeta...@gmail.com` on 11 Mar 2014 at 12:35

**Comment by GoogleCodeExporter** [comment]
```
Just installed it, now it works.

thanks a lot!
```

Original comment by `CHericks...@gmail.com` on 11 Mar 2014 at 12:52
